database/scripts/20220609_141721_inicial_up.sql

```diff
@@ -0,0 +1,83 @@
+-- Database generated with pgModeler (PostgreSQL Database Modeler).
+-- pgModeler version: 0.9.4
+-- PostgreSQL version: 12.0
+-- Project Site: pgmodeler.io
+-- Model Author: ---
+
+-- Database creation must be performed outside a multi lined SQL file. 
+-- These commands were put in this file only as a convenience.
+-- 
+-- object: udistrital_financiera | type: DATABASE --
+-- DROP DATABASE IF EXISTS udistrital_financiera;
+-- CREATE DATABASE udistrital_financiera;
+-- ddl-end --
+
+
+-- object: ingresos | type: SCHEMA --
+-- DROP SCHEMA IF EXISTS ingresos CASCADE;
+CREATE SCHEMA ingresos;
+-- ddl-end --
+
+SET search_path TO pg_catalog,public,ingresos;
```

Comment:
Revisar si con lo siguiente sigue funcionando:
```suggestion
SET search_path TO public,ingresos;
```